### PR TITLE
bounty_creation_redesign

### PR DESCRIPTION
**Pull request by AmitPaliwal-rs**
start working on ticket #393.

### CODE REVIEW

File: frontend/app/src/form/index.tsx [hunk 1]
```diff
@@ -16,24 +16,84 @@ import { colors } from '../colors';
 const BountyDetailsCreationData = {
```
Comment (kevkevinpal) [inline frontend/app/src/form/index.tsx:16]: Do you think we can put this in a const file under the form folder

`./src/form/formConstants.ts`

File: frontend/app/src/form/index.tsx [hunk 2]
```diff
@@ -81,6 +142,12 @@ export default function Form(props: any) {
       case 3:
         setSchemaData(BountyDetailsCreationData.step_3);
         break;
+      case 4:
```
Comment (kevkevinpal) [inline frontend/app/src/form/index.tsx:145]: do you think we can use an array in the `BountyDetailsCreationData` so we don't need to keep changing this bit of code an instead just need to change the values?

File: frontend/app/src/form/index.tsx [hunk 3]
```diff
@@ -329,163 +391,242 @@ export default function Form(props: any) {
...
-                    <EuiText
-                      className="SubHeadingText"
-                      style={{
-                        marginBottom: schemaData.step === 1 ? '29px' : '37px'
-                      }}
-                    >
-                      {schemaData.sub_heading}
+                  <div className="TopContainer">
+                    <EuiText className="stepText">
+                      {`STEP ${schemaData.step}`} <span className="stepTextSpan"> / 5</span>
```
Comment (kevkevinpal) [inline frontend/app/src/form/index.tsx:396]: we can make that a .length value if we change BountyDetailsCreationData to be an array

File: frontend/app/src/form/index.tsx [hunk 4]
```diff
@@ -631,8 +757,7 @@ export default function Form(props: any) {
                         justifyContent: 'center',
                         alignItems: 'center',
                         marginTop: 20
-                      }}
-                    >
+                      }}>
```
[the remark this answers is not in the document]
Comment (AmitPaliwal-rs) [reply frontend/app/src/form/index.tsx:709]: It might be the issue, I'll have to look at this once.

File: frontend/app/src/form/index.tsx (hunk 1 above)
Comment (AmitPaliwal-rs) [reply frontend/app/src/form/index.tsx]: Yeah, I have already pushed.

File: frontend/app/src/form/index.tsx (hunk 2 above)
Comment (AmitPaliwal-rs) [reply frontend/app/src/form/index.tsx:62]: Sure, I'll look into this.

File: frontend/app/src/people/utils/language_label_style.ts [hunk 5]
```diff
@@ -290,3 +290,92 @@ export const colourOptions: readonly ColourOption[] = [
     color: '#FF8D85'
   }
 ];
+
+export const BountyDetailsCreationData = {
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/language_label_style.ts:294]: let try creating a new file in `./src/form/formConstants.ts` it doesn't make sense to have this data in the `language_label_style.ts`

Comment (kevkevinpal) [reply frontend/app/src/people/utils/language_label_style.ts:294]: oh wait I see why you put that there, form is in a whole separate folder than people

Comment (kevkevinpal) [reply frontend/app/src/people/utils/language_label_style.ts:294]: maybe put it in another file named `bountyCreationConstants.ts` in `frontend/app/src/people/utils`

**Review by kevkevinpal — APPROVED** [review]
looking good going to merge

**Pull request merged**